conf/citest/pipelines/DumpGenomes.json

```diff
@@ -0,0 +1,39 @@
+{
+    "reference_db": "mysql://ensro@mysql-ens-compara-prod-8:4618/jalvarez_citest_dump_genomes_99",
+    "reference_dir": "/hps/nobackup2/production/ensembl/jalvarez/genome_dumps",
+    "database_tests": {
+        "job": [
+            {
+                "test": "num_rows",
+                "args": {
+                    "variation": 0,
+                    "group_by": "analysis_id",
+                    "filter_by": "analysis_id > 1"
+                }
+            }
+        ],
+        "analysis_base": [
+            {
+                "test": "content",
+                "args": {
+                    "columns": ["analysis_id", "logic_name"]
+                }
+            }
+        ]
+    },
+    "files_tests": [
+        {
+            "test": "size",
+            "args": {
+                "variation": 0
+            }
+        },
+        {
+            "test": "size",
+            "args": {
+                "variation": 0.1,
```

Comment:
This test seems to contradict the previous one. If you want to showcase what CITest can do, I would suggest to put examples of every test in a new file example.json